### PR TITLE
Update sugarfixpermissions to be faster on slow filesystems (EFS, OSXFS, etc)

### PR DESCRIPTION
**Pull request by jhoffmann**
You might want to double check the paths, you had vendor/phpunit/phpunit/phpunit but if I recall it's actually vendor/phpunit/bin/phpunit.

I also updated 2 of the opcache configurations to use the file location given in the Dockerfile.

### CODE REVIEW

**Comment by esimonetti** [comment]
Hey John, thanks!

I double checked on the vendor repo and it seems to be correct: `Mango/sugarcrm/vendor/phpunit/phpunit$ ls -la phpunit`

Let me double check that everything is in working order, as I will have to push out new images with the changes. Odd that nobody else noticed before! Opcache in theory should not even work well without the correct blacklist file...

Oh well, thanks again!

**Comment by esimonetti** [comment]
I see, it was only for cron! That's why nobody noticed... cool

File: images/permissions/apps/sugarfixpermissions [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-if [ -e ./bin/sugarcrm ]
+#!/usr/bin/env bash
+
+DIR=/var/www/html
+
+# Only change ownership on files that aren't currently owned by sugar:sugar
+find ${DIR} -noleaf \( \! -user sugar -o \! -group sugar \) -print0 | xargs -0 -r chown sugar:sugar
+
+# Only change the mode of files/directories that are not 664/775
+find ${DIR} -noleaf -type d \! -perm 775 -print0 2> /dev/null | xargs -0 -r chmod 775 2> /dev/null
```
Comment (esimonetti) [inline images/permissions/apps/sugarfixpermissions]: Probably for these 2 lines we want to apply it only to ./sugar?

Comment (jhoffmann) [reply images/permissions/apps/sugarfixpermissions]: Yes sorry, I'm actually running in /var/www/html without the subfolder in my images and missed that conversion.

Comment (jhoffmann) [reply images/permissions/apps/sugarfixpermissions]: I'll open a new PR to fix that shortly.

Comment (esimonetti) [reply images/permissions/apps/sugarfixpermissions]: Hi John, don't worry, I've already merged it on the dev branch and modified it.

Now I am running into another issue that prevents it from merging it to master though, related to elasticsearch 62 permissions issues which does not fail locally but it fails on the build automation for some reason...

As soon as I pass the hurdle you will see it in master

Cheers!

**Pull request merged**